app/api/utils.py

```diff
@@ -479,3 +474,29 @@ def readinto(self, b):
                 return 0    # indicate EOF
 
     return io.BufferedReader(IterStream(), buffer_size=buffer_size)
+
+def parse_excel_csv_reader(reader):
+    columns = next(reader)
+    data = []
+    if len(columns) == 1 and columns[0] != 'text':
+        if len(data) >= settings.IMPORT_BATCH_SIZE: #pragma: no cover
+            yield data
+            data = []
+        data.append({'text': columns[0]})
+    for i, row in enumerate(reader, start=2):
+        if len(data) >= settings.IMPORT_BATCH_SIZE: #pragma: no cover
```

Comment:
We should be able to cover the import batch size case via `@override_settings(IMPORT_BATCH_SIZE=some-low-number)` as in `TestParser`. Any specifics/details why we have to disable this?

Reply:
Ahh I didn't know about that thanks! Just updated the tests.